examples/st7789_280x240_simpletest.py

```diff
@@ -14,37 +15,67 @@
 
 from adafruit_st7789 import ST7789
 
+# set the display rotation
+rotation = 90
+if rotation not in (0, 90, 180, 270):
+    raise ValueError("The value of rotation must be one of: 0, 90, 180, 270")
+
+# Display settings depending on the selected rotation
+# first value default setting for 1.69" with 0° and 180° rotation
+# second value default setting for 1.69" with 90° and 270° rotation
+width = 240 if rotation in (0, 180) else 280
+height = 280 if rotation in (0, 180) else 240
```

Comment:
I think the remaining ruff check issue is it wanting these tuples used in the if statements to be literal sets instead.
i.e.
```
height = 280 if rotation in {0, 180} else 240
```
Essentially just swap to curly braces instead of parens around them.